src/pages/components/table.mdx

```diff
@@ -165,6 +165,28 @@ import TruncateCode from '!!raw-loader!../../examples/table/TableTruncate.tsx';
   <Truncate />
 </CodeExample>
 
+### Draggable
+
+This functionality is implemented using [react-beautiful-dnd](https://github.com/atlassian/react-beautiful-dnd).
```

Comment:
Recommend dropping "functionality is implemented using..." (as the implementation makes this clear at a glance) for more instructional text. Something like "Drag and drop functionality can be applied, allowing you to reorder rows." (UXCS / @vsrixyz  needed :wink:)

Reply:
I've updated this to use the new descriptive, but similar to the comment above, I think naming the implementation libraries is important since consumers have to roll this themselves.

Similar to the "drag and drop file upload" pattern, naming https://github.com/react-dropzone/react-dropzone is the majority of the documentation.